package/gluon-web-model/javascript/gluon-web-model.js

```diff
@@ -219,6 +215,20 @@
 				parent.parentNode.style.display = (parent.options.length <= 1) ? 'none' : '';
 		}
 
+		var nodes = document.querySelectorAll('[data-exclusive-with]');
+		for (var i = 0, node; (node = nodes[i]) !== undefined; i++) {
+			var excusive_with = JSON.parse(node.getAttribute('data-exclusive-with'));
```

Comment:
I just noticed a typo
`exclusive_with` would be correct

Reply:
Thanks.